notebooks/128-openvino-tokenizers/128-opnevino-tokenizers.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>please update table of content according the latest changes in main.</p><p> </p><p>Formatting looks strange in preview, maybe better to use image or diagram instead of latex (latex visualization may looks vary depends from viewer)?</p><p> </p><p>&gt;&gt;You can check the GPT-4 tokenizer output <a href="https://platform.openai.com/tokenizer" target="_blank">here</a>.  </p><p>for me, from position of user how never think about otkenization, it looks not clear why I need to check it? Maybe some additional required (e.g. You can check the GPT-4 tokenizer result <a href="https://platform.openai.com/tokenizer" target="_blank">here</a> for better understanding principles of tokenizer work) </p><p> </p><p> </p><p>typo:</p><p>Some tasks only need a tokenizer, like text classification, named entity recognition, question answering, and feature extraction. On the other hand, for tasks such as text generation, chat, translation, and abstractive summarization, both a tokenizer and a detokenizer are required<s>.m</s></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>